scene/gui/graph_edit.cpp

```diff
@@ -1305,11 +1413,24 @@ GraphEdit::GraphEdit() {
 	v_scroll->connect("value_changed", callable_mp(this, &GraphEdit::_scroll_moved));
 
 	zoom = 1;
+	linear_zoom = false;
+	zoom_step = 1.2;
```

Comment:
These can go in the header now (and `zoom`).

Reply:
Indeed, moved. Though I am now curious if we can move default min and max calculations to the header as well:

```cpp
	zoom_min = (1 / Math::pow(zoom_step, 8));
	zoom_max = (1 * Math::pow(zoom_step, 4));
```

I've left it in the constructor for the time being.